### DISCUSSION TITLE
Welcome to CodeGlob Discussions!

### DISCUSSION THREAD

**Discussion by hamdivazim-archive** [hamdivazim/CodeGlob#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community and provide helpful tips or feature requests. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Be respectful with your suggestions.

  To get started, comment below with an introduction of yourself and anything you think could help the CodeGlob community :)

**Comment by MKM12345** [comment]
Sorry hamd, but could I get access to the code files?

**Reply by MKM12345** [reply]
Also part of the team in the Readme

**Reply by hamdivazim-archive** [reply]
I've directly messaged you back with an answer.
